### PR TITLE
jxls: initial-integration

### PR DESCRIPTION
**Pull request by onionpsy**

### CODE REVIEW

File: projects/jxls/build.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -eu
+# Copyright 2023 Google Inc.
```
Comment (jonathanmetzman) [inline projects/jxls/build.sh:2]: LLC

Comment (jonathanmetzman) [inline projects/jxls/build.sh:2]: LLC

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Comment by SoltauFintel** [comment]
Can't create file because Github returns status 500.

**Pull request merged**